src/test/java/com/artipie/asto/StorageMoveTest.java

```diff
@@ -37,10 +37,10 @@
  * @since 0.14
  */
 @ExtendWith(StorageExtension.class)
+@Timeout(2)
 public final class StorageMoveTest {
```

Comment:
@andreoss why is it necessary to increase timeout here from one second to two?

Reply:
@genryxy It seems that this test is flakes, sometimes it fails on GHA.